.github/workflows/stale.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale because it has been open for 60 days with no activity. If no comment has been made and the `lifecycle/stale` label is not removed, this issue will be closed in 15 days.'
-        close-issue-message: 'This issue will be closed because it has been open for 75 days with no activity. If you still have any question about this issue, feel free to reopen it or create a new one.'
+        close-issue-message: 'This issue will be closed because it has been open for 15 days with no activity. If you still have any question about this issue, feel free to reopen it or create a new one.'
```

Comment:
```suggestion
        close-issue-message: 'This issue will be closed because it has been stale for 15 days with no activity. If you still have any question about this issue, feel free to reopen it or create a new one.'
```

Reply:
Rest LGTM